orderable/models.py

```diff
@@ -52,6 +52,23 @@ def prev(self):
 
         return self.get_filtered_manager().before(self)
 
+    def validate_unique(self, exclude=None):
+        if self._is_sort_order_unique_together_with_something():
+            exclude = exclude or []
+            if 'sort_order' not in exclude:
+                exclude.append('sort_order')
+        return super().validate_unique(exclude=exclude)
```

Comment:
We're still supporting python 2 for the moment, so:

```suggestion
        return super(Orderable, self).validate_unique(exclude=exclude)
```

Reply:
Fail - I am so used to working only on Python 3 project code!